core/jazz_splunk-kinesis-log-streamer/components/utils.js

```diff
@@ -21,11 +21,78 @@
 	@version: 1.0
 **/
 
+const AWS = require('aws-sdk');
+const Uuid = require("uuid/v4");
+const request = require('request');
 const logger = require("../components/logger.js");
 const global_config = require("../config/global-config.json");
 const truncate = require('unicode-byte-truncate');
 
 // Helper functions
+
+function assumeRole(configData, serviceData){
+  var isPrimary, roleArn, nonSlsApp = false;
+  if(serviceData){
+  	isPrimary = checkIsPrimary(serviceData.deployment_accounts[0].accountId, configData);
+  	roleArn = getRolePlatformService(serviceData.deployment_accounts[0].accountId, configData);
+  } else {
+  	nonSlsApp = true
```

Comment:
When do hit this else?

Reply:
@SauarvD Can we go ahead and also make the log groups for all service type consistently delimited by `_`?

Reply:
Refactored the code.